elasticsearch-details/impersonation.md

```diff
@@ -151,4 +154,18 @@ After clicking add/edit user buttons(5), you will see a dialog with an option to
 
 All logs of impersonated user will have this format `[<log level>][plugins][ReadonlyREST][<filename>][impersonating <impersonated user username>]`
 
-![Impersonate user](<../.gitbook/assets/impersonate_impersonated_menu.png>)
\ No newline at end of file
+![Impersonate user](<../.gitbook/assets/impersonate_impersonated_menu.png>)
+
+## Impersonation limitations
+
+Impersonation mode has some limitations. Please check if they have an impact on your use cases:
+
+* Not all features available in the ROR configuration are testable with impersonation mode.
+Some rules used in ROR ACL do not support impersonation. For example, auth rule with hashed credentials(e.g. `auth_key_sha512`) can be used in impersonation mode only when credentials follow the format `USER_NAME: HASH(PASSWORD)`; A fully hashed username and password don't allow fetching a username. The auth rule in such a format won't match during impersonation.
+A list of the rules with info about impersonation support can be found [here](../elasticsearch.md#rules).
+
+* Test settings are stored in the memory of the node that handled the POST tests settings request.
```

Comment:
hmmm... what if you define multiple ES nodes in Kibana? Then the experience will be inconsistent?

Reply:
Yes, it looks like the experience will be inconsistent. In the next tasks, we will introduce distributed test settings. But for now, how to inform users about that? I think it's worth informing them about that.

Reply:
Yes absolutely, it's a very important detail. Btw I remember now we already talked about this 👍